keps/sig-auth/3331-structured-authentication-configuration/README.md

```diff
@@ -724,49 +662,40 @@ providers such as Okta, Azure AD, etc:
 
 #### Beta
```

Comment:
Do we have a test that checks that distributed claims (for groups) work with CEL validation?

Reply:
Moved this to GA criteria based on the discussion.